docs/man_pages/lib-management/kendoui-install.md

```diff
@@ -0,0 +1,51 @@
+kendoui install
+==========
+
+Usage | Synopsis
+------|-------
+List all versions | `$ appbuilder kendoui install [--verified] [--latest]`
```

Comment:
We didn't have on `update-kendoui` and I decided not to change the api that's why.
The command can still be used in a non-interactive console with `--latest` flag however